dot/sync/worker.go

```diff
@@ -76,6 +76,7 @@ type worker struct {
 	startNumber  *big.Int
 	targetHash   common.Hash
 	targetNumber *big.Int
+	pendingBlock *pendingBlock // if this worker is tied to a specific pending block, this field is set
```

Comment:
Less work for #2051 :smile_cat: 

```suggestion
	// if this worker is tied to a specific pending block, this field is set
	pendingBlock *pendingBlock
```